cmd/cloud.go

```diff
@@ -330,7 +333,8 @@ This will execute the test on the k6 cloud service. Use "k6 login cloud" to auth
 				return ExitCode{error: errors.New("Test progress error"), Code: cloudFailedToGetProgressErrorCode}
 			}
 
-			fprintf(stdout, "     test status: %s\n", ui.ValueColor.Sprint(testProgress.RunStatusText))
+			valueColor := getColor(noColor || !stdoutTTY, color.FgCyan)
```

Comment:
Given how now you repeat `color.FgCyan` a lot I would argue it should probably be a constant somewhere ... probably named `valueColorColor` :P

Reply:
It's already a constant :stuck_out_tongue: It's used in like 5 places across 2 modules, and not always for values (e.g. the banner), so I don't want to define a constant for that.